system/BaseModel.php

```diff
@@ -516,7 +516,7 @@ abstract public function chunk(int $size, Closure $userFunc);
      */
     public function find($id = null)
     {
-        $singleton = is_numeric($id) || is_string($id);
+        $singleton = is_numeric($id) || \is_string($id);
```

Comment:
Did it miss this because there were two on a line?

Reply:
Based here, `is_numeric` is not optimized.

https://github.com/php/php-src/blob/3a599dccb12395f9b9647dd5cd9f00322b9b644a/Zend/zend_compile.c#L3886-L3909